package.json

```diff
@@ -3,8 +3,8 @@
   "license": "MIT",
   "scripts": {
     "start": "node ./lib/run.js",
-    "test": "jest --coverage --runInBand --forceExit",
-    "test:watch": "jest --runInBand --forceExit --watch --notify",
+    "test": "node --expose-gc --max-old-space-size=4096 ./node_modules/.bin/jest --coverage --runInBand --logHeapUsage --forceExit",
+    "test:watch": "node --expose-gc --max-old-space-size=4096 ./node_modules/.bin/jest --runInBand --logHeapUsage --forceExit --watch --notify",
```

Comment:
I _believe_ `node_modules/.bin` are automatically part of the `$PATH` thanks to some node/package.json magic, so we might be able to drop that? Buuut I have never done anything this low level with node, so it's very possible that I'm very wrong here 😅

Reply:
You are correct about the PATH, but we don't want to invoke jest, because that would use it's 'shebang' line which is running `node` without any flags. So that is, what we used before. 
The purpose of this change is to directly call the `node` executable and point it to the script it should run. It this case `jest`. So the PATH does only matter for the first command in the line (which is node). Hope that makes sense.